tsdoc/src/details/StandardTags.ts

```diff
@@ -72,6 +88,20 @@ export class StandardTags {
     standardization: Standardization.Extended
   });
 
+  /**
+   * (Core)
+   *
+   * This block tag communicates that an API item is no longer supported and may be removed
+   * in a future release.  The `@deprecated` tag is followed by a sentence describing
+   * the recommended alternative.  It recursively applies to members of the container.
+   * For example, if a class is deprecated, then so are all of its members.
+   */
+  public static readonly deprecated: TSDocTagDefinition = StandardTags._defineTag({
```

Comment:
(the `@deprecated` definition isn't changed by this PR; I merely fixed the alphabetical ordering)